web/packages/teleterm/src/ui/Vnet/useVnetLauncher.tsx

```diff
@@ -53,14 +37,12 @@ export const useVnetAppLauncher = (): {
    * If the user is yet to start VNet, it opens the info doc. If they already started it in the past,
    * it starts VNet and then copies the address of the app to the clipboard.
```

Comment:
```suggestion
   * it starts VNet and then copies the address of the resource to the clipboard.
```

Below line 34 we can add another bullet point mentioning clicking "Connect with VNet" next to an SSH node.